docs/spec/QueueStatus.schema.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "QueueStatus",
+  "type": "object",
+  "properties": {
+    "tasks": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "key": {"type": "string"},
+          "status": {
+            "enum": ["queued", "running", "success", "error", "orphan", "interrupted", "redirect", "missing"]
+          },
+          "name": {"type": "string"},
+          "version": {
+            "type": [ "string", "null" ]
+          }
+        },
+        "required": ["key", "status", "name", "version"],
```

Comment:
Presumably this spec is inherited from orderly.server but `version` being `required` doesn't seem consistent with `spec.md` or the tests below